tests/syntax-tests/source/Swift/test.swift

```diff
@@ -0,0 +1,268 @@
+class Person {
+// We can define class property here
+var age  = 25
+// Implement Class initializer. Initializers are called when a new object of this class is created
+init() { 
+   print(“A new instance of this class Person is created.”) 
+ } 
+} 
+// We can now create an instance of class Person - an object - by putting parentheses after the class name
+let personObj =  Person()
+// Once an instance of Person class is created we can access its properties using the dot “.” syntax.
+print(“This person age is \(personObj.age)”)
+
+import Foundation
+class Friend : Comparable {
+    let name : String
+    let age : Int
+    
+    init(name : String, age: Int) {
+        self.name = name
+        self.age = age
+    }
+}
+func < (lhs: Friend, rhs: Friend) -> Bool {
+    return lhs.age < rhs.age } func > (lhs: Friend, rhs: Friend) -> Bool {
+    return lhs.age > rhs.age
+}
+func == (lhs: Friend, rhs: Friend) -> Bool {
+    var returnValue = false
+    if (lhs.name == rhs.name) && (lhs.age == rhs.age)
+    {
+        returnValue = true
+    }
+    return returnValue
+}
+
+  let friend1 = Friend(name: "Sergey", age: 35)
+        let friend2 = Friend(name: "Sergey", age: 30)
+        
+        print("\Compare Friend object. Same person? (friend1 == friend2)")
```

Comment:
what is `\C`?

Reply:
Oo sorry, there is a typo.
I will correct it.